src/modules/attendees/attendees.service.ts

```diff
@@ -105,12 +133,14 @@ export class AttendeesService {
                             ),
                         );
                 }
+                this.logger.log(apiId, 'Attendees successfully registered')
```

Comment:
@souravbhowmik1999  Please add correct msg. here like `Successfully get attendees for this eventid : event id`